app/views/user_informations/_form.html.erb

```diff
@@ -61,10 +66,62 @@
     <%= f.label :city %><br>
     <%= f.text_field :city, :title => 'Enter Name of City', :class => 'tooltips'  %>
   </div>
-  <div class="field">
+ <div class="field">
     <%= f.label :state %><br>
-    <%= f.text_field :state, :title => 'Enter Name of State', :class => 'tooltips'  %>
-  </div>
+    <%= f.select :state, {}, { include_blank: 'My State' }, :title => 'Select the state in which you legally reside', :class => 'form-control tooltips' do  %>
```

Comment:
You could also use the [countries gem](https://github.com/hexorx/countries) to pull out a more complete list (has Armed Forces, Guam, Virgin Islands, etc.) with less code. If we only want the 50 contiguous states, and we aren't using this anywhere else, the gem is overkill.